backend/index.js

```diff
@@ -0,0 +1,27 @@
+import http from "http";
```

Comment:
This is a good start, but we'll soon need an ORM and something to handle database migrations

Reply:
I'd like to split that into a separate ticket - and we'll use something like Express (or Fastify?) instead of a basic node server 👍 

Mostly wanted to get the base boilerplate stuff set up so we'll all have a common starting point for adding to it. I'm about to push some Eslint tweaks, then I'll merge it and we can create some specific tickets to tackle